testing/sdk_marathon.py

```diff
@@ -225,16 +225,3 @@ def bump_task_count_config(service_name, key_name, delta=1):
     updated_node_count = int(config['env'][key_name]) + delta
     config['env'][key_name] = str(updated_node_count)
     update_app(service_name, config)
-
-
-def get_mesos_api_version(service_name):
```

Comment:
Note that these `testing/` files are used in a number of other repos -- including ones not controlled by us. Is this tooling change "safe" for all of those too? We may want to leave these in here for the time being and mark them as deprecated.

Reply:
ya that makes sense, we can keep the tooling functions, will update